### ISSUE TITLE
不理解DAN中SGD的学习率以及MMD前的系数lambd

### ISSUE THREAD

**Issue by qq490516306** [jindongwang/transferlearning#135]
作者你好，我想问下DAN代码中
LEARNING_RATE = lr / math.pow((1 + 10 * (i - 1) / (iteration)), 0.75)，
这个初始的学习率为啥要这样的选取，有啥优点嘛？

还有SGD优化器用adam效果会有影响吗，以及SGD里的学习率参数为啥要设置成分类层是特征提取层的十倍。
optimizer = torch.optim.SGD([
        {'params': model.sharedNet.parameters()},
        {'params': model.cls_fc.parameters(), 'lr': LEARNING_RATE},
        ], lr=LEARNING_RATE / 10, momentum=momentum, weight_decay=l2_decay)

还有后面的这个MMD损失前面的系数有啥意义？
lambd = 2 / (1 + math.exp(-10 * (i) / iteration)) - 1
        loss = cls_loss + lambd * mmd_loss

**Comment by jindongwang** [comment]
Lr的设定和10倍关系被很多方法采用，算是一个经验。你可以去看DDC的文章，里面讲了他用10倍lr；这个奇怪的lr衰减的式子你可以去看DANN，是他们提出来的。
SGD你用任何方法都可以，不局限于SGD。
两个loss相加，当然要搞一个平衡的系数来权衡两部分的重要性呀。
